test/parallel/test-http-exceptions.js

```diff
@@ -20,8 +20,13 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 'use strict';
-require('../common');
+const common = require('../common');
+const Countdown = require('../common/countdown');
 const http = require('http');
+const NUMBER_OF_EXCEPTIONS = 4;
+const countdown = new Countdown(NUMBER_OF_EXCEPTIONS, () => {
+  process.exit(0);
```

Comment:
I think this should be `server.close` instead? (of course this line would need to be placed after the server creation)

Reply:
@joyeecheung calling `server.close()` does not close the node process for some reason. resulting in a `--- TIMEOUT ---` in the tests.